dds/DCPS/unique_ptr.h

```diff
@@ -132,7 +132,7 @@ class unique_ptr {
   }
 
 private:
-  unique_ptr(const unique_ptr&);
+  unique_ptr(const unique_ptr&) {};
```

Comment:
`unique_ptr` is not copyable, so the copy constructor is deliberately declared but not defined.

Reply:
Seems that the modification is not needed anymore to link C++/CLI projects. I would say it was fixed on this commit 8b02ca2